moped-editor/src/views/dashboard/DashboardView.js

```diff
@@ -100,19 +102,142 @@ const TABS = [
   },
 ];
 
+const UserGreeting = ({ classes, userName }) => {
+  const date = new Date();
+
+  /** Build custom user greeting
+   */
+  return (
+    <Grid className={classes.greeting}>
+      <Typography className={classes.greetingText}>
+        <strong>{`Good ${getTimeOfDay(date)}, ${userName}!`}</strong>
+      </Typography>
+      <Typography variant="h1" className={classes.date}>
+        {getCalendarDate(date)}
+      </Typography>
+    </Grid>
+  );
+};
+
+/** Hook that provides memoized column settings */
+const useColumns = ({ refetch, handleSnackbar, classes }) =>
+  useMemo(() => {
+    return [
+      {
+        headerName: "ID",
+        field: "project",
+        valueGetter: (params) => {
+          return params.project_id;
+        },
+        editable: false,
+        flex: 0.5,
+      },
+      {
+        headerName: "Full name",
+        field: "project_name_full",
+        editable: false,
+        renderCell: ({ row }) => (
+          <Link
+            href={`/moped/projects/${row.project.project_id}`}
+            sx={{
+              overflow: "hidden",
+              textOverflow: "ellipsis",
+              display: "block",
+            }}
+          >
+            {row.project.project_name_full}
+          </Link>
```

Comment:
There is a small regression here since the MUI Link is using browser redirect instead of client-side navigation with React Router. Could you update the `Link` component to use the `component` and `to` props like in [this project list view column](https://github.com/cityofaustin/atd-moped/blob/c79b0ce8476e2720e985e9ecca521f753851e76f/moped-editor/src/views/projects/projectsListView/helpers.js#L162-L163)? 🙏